### PR TITLE
Add locales for keyboard shortcuts

### PR DESCRIPTION
**Pull request by ben-lu-uw**
For https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/683

### CODE REVIEW

**Comment by prushforth** [comment]
@Malvoz if you wouldn't mind looking over how I translated to swedish, that would be great, thank you! 🙏 
1

File: src/_locales/sv/messages.json [hunk 1]
```diff
@@ -54,5 +54,38 @@
...
+    "message": "Navigationsnycklar för geografiska egenskaper"
+  },
+  "kbdPanUp": {
+    "message": "Panorera upp"
+  },
+  "kbdPanDown": {
+    "message": "Panorera ner"
+  },
+  "kbdPanLeft": {
+    "message": "Panorera till vänster"
```
Comment (Malvoz) [inline src/_locales/sv/messages.json:74]:
```suggestion
    "message": "Panorera vänster"
```

I think I prefer "Pan left", as opposed to "Pan to left".

File: src/_locales/sv/messages.json [hunk 2]
```diff
@@ -54,5 +54,38 @@
...
+  },
+  "kbdPanRight": {
+    "message": "Panorera till höger"
```
Comment (Malvoz) [inline src/_locales/sv/messages.json:77]:
```suggestion
    "message": "Panorera höger"
```

I think I prefer "Pan right", as opposed to "Pan to right".

File: src/_locales/sv/messages.json [hunk 3]
```diff
@@ -54,5 +54,38 @@
...
+  },
+  "kbdPanIncrement": {
+    "message": "Ökning för panorering av kartan"
```
Comment (Malvoz) [inline src/_locales/sv/messages.json:80]:
```suggestion
    "message": "Öka panorering"
```

This roughly translates to "The increase for panning of maps". The context is already a map, so no need to mention that IMO.

File: src/_locales/sv/messages.json [hunk 4]
```diff
@@ -54,5 +54,38 @@
...
+  },
+  "kbdZoom": {
+    "message": "Zooma in/ut 3 nivåer"
+  },
+  "kbdPrevFeature": {
+    "message": "Föregående geografisk egenskap"
```
Comment (Malvoz) [inline src/_locales/sv/messages.json:86]:
```suggestion
    "message": "Föregående geografiskt objekt"
```

"Egenskap(er)" translates to "attribute(s)" or "characteristic(s)", I think "objekt" (object in English) is a better translation.

File: src/_locales/sv/messages.json [hunk 5]
```diff
@@ -54,5 +54,38 @@
...
+  },
+  "kbdNextFeature": {
+    "message": "Nästa geografisk egenskap"
```
Comment (Malvoz) [inline src/_locales/sv/messages.json:89]:
```suggestion
    "message": "Nästa geografiskt objekt"
```

Same as https://github.com/Maps4HTML/mapml-extension/pull/43/files#r945659273.

File: src/_locales/sv/messages.json [hunk 6]
```diff
@@ -54,5 +54,38 @@
...
     "message": "Visa i helskärm"
+  },
+  "kbdShortcuts": {
+    "message": "Tangentbordsgenvägar"
+  },
+  "kbdMovement": {
+    "message": "Rörelsenycklar"
+  },
+  "kbdFeature": {
+    "message": "Navigationsnycklar för geografiska egenskaper"
```
Comment (Malvoz) [inline src/_locales/sv/messages.json:65]:
```suggestion
    "message": "Navigationsnycklar för geografiska objekt"
```

Similar to https://github.com/Maps4HTML/mapml-extension/pull/43#pullrequestreview-1072549067.

File: src/_locales/sv/messages.json [hunk 7]
```diff
@@ -54,5 +54,38 @@
   },
   "btnFullScreen": {
     "message": "Visa i helskärm"
+  },
+  "kbdShortcuts": {
+    "message": "Tangentbordsgenvägar"
```
Comment (Malvoz) [inline src/_locales/sv/messages.json:59]:
```suggestion
    "message": "Kortkommandon"
```

I believe the norm is to use "Kortkommandon", which directly translates to "short commands".

File: src/_locales/sv/messages.json [hunk 8]
```diff
@@ -54,5 +54,38 @@
...
+  },
+  "kbdMovement": {
+    "message": "Rörelsenycklar"
```
Comment (Malvoz) [inline src/_locales/sv/messages.json:62]:
```suggestion
    "message": "Rörelsekommandon"
```

**Pull request merged**